src/Importify/Resolution.hs

```diff
@@ -20,16 +23,23 @@ import           Importify.Common                   (Identifier (..),
                                                      importSpecToIdentifiers)
 
 symbolByName :: String -> [N.Symbol] -> Maybe N.Symbol
-symbolByName name symbols = head $ do
-    symbol <- symbols
-    guard $ symbolName symbol == stringToName name
-    pure symbol
+symbolByName name =
+    listToMaybe . filter ((stringToName name ==) . symbolName)
```

Comment:
I think, it can be refactored better, if we put this function near other utilities, but that's minor

`findBy :: (a -> Bool) -> [a] -> Maybe a`